### ISSUE TITLE
Im getting  Unable to map in all my apps.

### ISSUE THREAD

**Issue by titusfx** [ngx-builders/source-map-analyzer#35]
`ng run canvas-sh:analyze`

```bash
Your global Angular CLI version (9.1.4) is greater than your local
version (9.0.5). The local Angular CLI version is used.

To disable this warning use "ng config -g cli.warnings.versionMismatch false".
93% after chunk asset optimization SourceMapDevToolPlugin styles.dddc646dfe9dc8a6a293% after chunk asset optimization SourceMapDevToolPlugin styles-es2015.7afbc82558e93% after chunk asset optimization SourceMapDevToolPlugin runtime-es2015.0811dcefd393% after chunk asset optimization SourceMapDevToolPlugin main-es2015.779165b467b0a93% after chunk asset optimization SourceMapDevToolPlugin polyfills-es2015.8631121f93% after chunk asset optimization SourceMapDevToolPlugin polyfills-es5.22892daab4293% after chunk asset optimization SourceMapDevToolPlugin styles.dddc646dfe9dc8a6a293% after chunk asset optimization SourceMapDevToolPlugin styles-es2015.7afbc82558eGenerating ES5 bundles for differential loading...
ES5 bundle generation complete.

chunk {0} runtime-es2015.0811dcefd377500b5b1a.js, runtime-es2015.0811dcefd377500b5b1a.js.map (runtime) 1.51 kB [entry] [rendered]
chunk {0} runtime-es5.0811dcefd377500b5b1a.js, runtime-es5.0811dcefd377500b5b1a.js.map (runtime) 1.51 kB [entry] [rendered]
chunk {2} polyfills-es2015.8631121f3a5ed024da5c.js, polyfills-es2015.8631121f3a5ed024da5c.js.map (polyfills) 35.7 kB [initial] [rendered]
chunk {3} polyfills-es5.22892daab42a4c9e5aa9.js, polyfills-es5.22892daab42a4c9e5aa9.js.map (polyfills-es5) 128 kB [initial] [rendered]
chunk {1} main-es2015.779165b467b0aa17f96d.js, main-es2015.779165b467b0aa17f96d.js.map (main) 266 kB [initial] [rendered]
chunk {1} main-es5.779165b467b0aa17f96d.js, main-es5.779165b467b0aa17f96d.js.map (main) 322 kB [initial] [rendered]
chunk {4} styles.dddc646dfe9dc8a6a281.css, styles.dddc646dfe9dc8a6a281.css.map (styles) 0 bytes [initial] [rendered]
Date: 2020-06-07T09:55:21.658Z - Hash: 0c46ce2f1e2cd2d7123c - Time: 156865ms
dist/canvas-sh/main-es2015.779165b467b0aa17f96d.js

  Unable to map 130/272617 bytes (0.05%)

```
